website/docs/components/alert/partials/code/how-to-use.md

```diff
@@ -43,6 +43,29 @@ Optionally, you can pass only `title` or only `description`.
 </Hds::Alert>
 ```
 
+### Tag
+
+The `tag` value for the Alert Title changes the HTML element wrapping the title content. When organizing the content on a webpage, the heading levels should reflect the structure of the page. For example, if an Alert appears directly below the main heading of the page, it should be `"h2"`.
+
+```handlebars
+<div class="doc-alert-flex-layout">
```

Comment:
Question: I added a new `alert.scss` file to add a margin below the heading, but for some reason the styles aren't being applied on the page. Am I missing a step for how to add styles to the component examples?

Reply:
Preview where you can see the margin isn't added: https://hds-website-qsdgf6qnj-hashicorp.vercel.app/components/alert?tab=code#tag

Reply:
ahh, ty @Dhaulagiri !